mhlo_builder/csrc/mlir/mhlo/builder/standard.cpp

```diff
@@ -177,7 +177,7 @@ llvm::Optional<mlir::Value> BuildCastStdConstScalarToHloConstTensor(
     return llvm::None;
   }
   const mlir::Attribute& val_attr = def.getValue();
-  auto scalar_ty = mlir::RankedTensorType::get({}, val_attr.getType());
+  auto scalar_ty = mlir::RankedTensorType::get({}, std_scalar.getType());
```

Comment:
any difference here?

Reply:
llvm is updated, the old api is not valid here